scripts/generate_dictionary_synonyms_test_epub.py

```
@@ -0,0 +1,145 @@
+        # META-INF/container.xml
+        epub.writestr('META-INF/container.xml', '''<?xml version="1.0"?>
+<container version="1.0" xmlns="urn:oasis:names:tc:opendocument:xmlns:container">
+  <rootfiles>
+    <rootfile full-path="OEBPS/content.opf" media-type="application/oebps-package+xml"/>
+  </rootfiles>
+</container>''')
+
+        # Build spine and manifest
+        manifest_items = []
+        spine_items = []
+        for i, (chapter_title, content) in enumerate(chapters):
```

Ruff (0.15.21)

[warning] Loop control variable `chapter_title` not used within loop body

(B007)